cronos-play/getting-started_unreal/quick-start/wallet.md

```diff
@@ -0,0 +1,131 @@
+# Wallet
+
+Most of the wallet functionality is insecure / meant for development testing. To connect Wallets with Dapps/Dgames securely, we bring the open communications protocol for web3 into Unreal Engine. Please check [WalletConnect](walletconnect.md).
+
+### InitializeWallet
+
+Restore wallet with mnemonics.
+
+<figure><img src="../../../.gitbook/assets/image (14) (1).png" alt=""><figcaption></figcaption></figure>
+
+* `Mnemonics`: mnemonics. to restore
+* `Password`: salt in mnemonics restoration
+* `Output`: generated address (index=0)
+* `Success`: whether succeed or not
+* `Output Message`: error message, "" if succeed
+
+## InitializeNewDevelopmentOnlyWallet
```

Comment:
What's the difference between InitializeWallet and InitializeNewDevelopmentOnlyWallet?

Reply:
InitializeWallet = Restore wallet with mnemonics and password
InitializeNewDevelopmentOnlyWallet = Create a new wallet with password and wordcount

Reply:
Documents in source code are incorrect, I'll fix them.

Reply:
I think we need to add some extra descriptions

Reply:
Done